test/allowlist_for_publicAPI.json

```diff
@@ -130,14 +130,15 @@
     "Node",
     "Optional",
     "QConfigAny",
+    "QConfigMapping",
```

Comment:
No new entry should be added to this list!
Make sure to properly setup your module to reflect what is public and what is not public.

Reply:
I remember last time I tried there were some problems with this test, wondering if this is fixed now or not

Reply:
Yes, this test has been working fine for the past month now.

Reply:
Ok, I removed the changes in this file. Please have another look @albanD.